client/src/Tetromino.ts

```diff
@@ -123,6 +123,25 @@ export class Tetromino {
         };
     }
 
+    static createFromState(
+        state: TetrominoState,
+        ccRotations: number
+    ): Tetromino {
+        const tetromino = new Tetromino(state.type);
+        Tetromino.updateFromState(tetromino, state, ccRotations);
+        return tetromino;
+    }
```

Comment:
With the refactor of on playerPlaced, this is now unused. Do we have a known use for it in the future?

Reply:
Also this is the wrong place to put it, but maybe we should also update `reportPosition` to be `reportState` so it falls in line with the new terminology